ex2_system/source/beacon/beacon_task.c

```diff
@@ -117,6 +64,19 @@ static void *beacon_daemon(void *pvParameters) {
         scw[5] = 1;
         HAL_UHF_setSCW(scw);
 
+        //Send second beacon packet
+        memcpy(&(beacon_msg.message), &beacon_packet_two, sizeof(beacon_packet_2_t));

```

Comment:
Should there be a delay between the beacon messages to give it a chance to send the beacon?

Reply:
good point